{
  return toException(cause.getMessage(),cause);
}
